Remotion/Validation/Globalization.UnitTests/LocalizedValidationMessageFactoryTest.cs

```diff
@@ -736,6 +736,34 @@ public void CreateValidationMessageForPropertyValidator_WithDecimalValidator_Ret
       Assert.That(validationMessage.ToString(), Does.StartWith("Enter a number with no more than {0} integer digits and {1} decimal places."));
     }
 
+    [Test]
+    [SetUICulture("de-CH")]
+    public void CreateValidationMessageForPropertyValidator_WithGreaterThanValidator_ReturnsLocalizedValidationMessage_UsingSpecificUICulture ()
+    {
+      _propertyStub.Setup(_ => _.PropertyType).Returns(typeof(object));
+      var validator = new GreaterThanValidator(5, _validationMessageStub.Object);
+      var validationMessage = _factory.CreateValidationMessageForPropertyValidator(validator, _propertyStub.Object);
+
+      Assert.That(validationMessage, Is.Not.Null);
+      Assert.That(validationMessage, Is.InstanceOf<ResourceManagerBasedValidationMessage>());
+
+      Assert.That(validationMessage.ToString(), Is.EqualTo("Geben Sie einen Wert grösser als \"{0}\" ein."));
+    }
+
+    [Test]
+    [SetUICulture("de-CH")]
+    public void CreateValidationMessageForPropertyValidator_WithInclusiveRangeValidator_ReturnsLocalizedValidationMessage_UsingGeneralUICultureAsFallBack ()
+    {
+      _propertyStub.Setup(_ => _.PropertyType).Returns(typeof(object));
+      var validator = new InclusiveRangeValidator(5, 10, _validationMessageStub.Object);
+      var validationMessage = _factory.CreateValidationMessageForPropertyValidator(validator, _propertyStub.Object);
+
+      Assert.That(validationMessage, Is.Not.Null);
+      Assert.That(validationMessage, Is.InstanceOf<ResourceManagerBasedValidationMessage>());
+
+      Assert.That(validationMessage.ToString(), Is.EqualTo("Geben Sie einen Wert von \"{0}\" bis \"{1}\" ein."));
+    }
+
```

Comment:
Please revert the tests. We typically do not test the different localizations via unit tests because it creates a maintainance dependency, in particular for non-English texts.

We do sometimes have unit tests that must work with the default culture texts because we a) want to test that the placeholders are filled correctly or b) the correct resource is resolved or c) it's easier for reason. 

For any other language, specific unit tests which test that the locatization has been created correctly does not scale. It's a) a copy&paste operation of potentially longer texts in a language we don't know so it's much harder to verify and b) localizations are updated on demand if there's typos found or user feedback. Needing to hunt down the correct test and fixing the string by also pasting it in the test does not add value.

So, instead we rely on tooling support (e.g. Rider/Resharper's resource editor) and simple review to make sure the changes are correct (no extra placeholders, the correct value was pasted, etc)